src/components/tabs/tabs.scss

```diff
@@ -0,0 +1,49 @@
+@use "../../settings/constants";
+
+.mykn-tabs {
+  &__buttons {
+    background-color: var(--theme-shade-50);
+    border: 1px solid var(--theme-shade-100);
+    width: fit-content;
+    display: flex;
+    align-items: center;
+    gap: 17px;
+    padding: 6px 40px;
+    border-radius: var(--border-radus-s);
+    flex-wrap: wrap;
+  }
+
+  &__button {
+    &--active {
+      background-color: var(--theme-color-primary-100);
+      color: var(--theme-color-primary-800);
+    }
+  }
+
+  .mykn-button {
+    &--variant-transparent {
+      &:hover {
+        background-color: var(--theme-color-primary-100);
```

Comment:
Nice catch of the transparent button in hover state, I think this is the correct styling (and I probably picked the wrong values), can you update the main button stylesheet and remove this code here so that all transparent button will have a consistent (and proper) styling?

The new lines in the `button.scss` should be:

```
--mykn-button-color-background: var(--theme-color-primary-100);
--mykn-button-color-text: var(--theme-color-primary-800);
```

If we don't need the specific override, we don't need the changes for the additional classname for the button so I guess we can remove that as well (to discourage too many css overrides).